app/views/changeset/_jira_issues.html.erb

```diff
@@ -1,7 +1,7 @@
-<% if jira_issues.any? %>
+<% if changeset.jira_issues.any? %>
```

Comment:
this assumes `jira_issues` is cached ... could avoid that assumption and cache-logic by using `if jira_issues = changeset.jira_issues.presence`